Firmware/mmu2.cpp

```diff
@@ -1048,7 +1054,7 @@ void MMU2::OnMMUProgressMsgSame(ProgressCode pc) {
             case FilamentState::NOT_PRESENT:
                 // fsensor not triggered, continue moving extruder
                 if (!planner_any_moves()) { // Only plan a move if there is no move ongoing
-                    MoveE(2.0f, MMU2_LOAD_TO_NOZZLE_FEED_RATE);
+                    MoveE(350.0f, MMU2_LOAD_TO_NOZZLE_FEED_RATE);
```

Comment:
Please add the note about the long extrusion prevention here.

Reply:
Thanks I forgot about it

Reply:
Done